solr_wrapper.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.7"
   spec.add_development_dependency "rake", "~> 10.0"
 
+  spec.add_development_dependency "pry"
```

Comment:
Remove this commit. I'm not interested in this gem requiring `pry` for development.